script.js

```diff
@@ -1,4 +1,98 @@
+//Base
+// Requirements
+// There are 2 players and players take turns.
+// When a player clicks Submit, the game rolls 2 dice and shows the dice rolls, for example 3 and 6.
+// The player picks the order of the dice they want. For example, if they wanted the number 63, they would specify that the 2nd dice goes first. You can choose how the player specifies dice order.
+// After both players have rolled and chosen dice order, the player with the higher combined number wins.
+var gameDiceRoll = "Game Dice Roll";
+var diceSelect = "Dice Selection";
+var gameMode = gameDiceRoll;
+var winner = "Determine the Winner";
+
+var currPlayer = 1;
+
+var diceRolled = [];
+var player1Results = "";
+var player2Results = "";
+
+var rollDice = function () {
+  var diceNum = Math.floor(Math.random() * 6 + 1);
+  return diceNum;
+};
+
+var arrayToNum = function (num1, num2) {
+  return Number(String(num1) + String(num2));
+};
```

Comment:
This is a really nice and useful function. Which you have Wayne to thank for : )